tests/rust-plugins/swc_internal_plugin/Cargo.toml

```diff
@@ -5,16 +5,18 @@ edition = "2021"
 name = "swc_internal_plugin"
 publish = false
 version = "0.1.0"
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [lib]
-crate-type = ["rlib", "cdylib"]
+crate-type = ["cdylib"]
+
+# swc_atoms -> string_cache -> parking_lot -> https://github.com/Amanieu/parking_lot/issues/269
```

Comment:
this'll be some headache if this won't be resolved in our dep tree in the future when we need to stablize scaffolding template.